### PR TITLE
AP-724 Add Dependant in full time education or training page

### PR DESCRIPTION
**Pull request by reggieb**
[Jira AP-724](https://dsdmoj.atlassian.net/browse/AP-724)

Adds a page to the Citizen journey with forms etc.

### CODE REVIEW

File: app/services/flow/flows/citizen_dependants.rb [hunk 1]
```diff
@@ -18,6 +18,13 @@ class CitizenDependants < FlowSteps
         },
         other_dependant: {
           path: '[PLACEHOLDER] do you have any other dependant?'
+        },
+        dependants_full_time_educations: {
+          # path: ->(_) { urls.citizens_dependant_full_time_education_path },
+          forward: ->(_) { :dependants_incomes }
```
Comment (Kalagan) [inline app/services/flow/flows/citizen_dependants.rb]: you need to pass the `dependant` here

File: spec/requests/citizens/dependants/full_time_educations_spec.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'does not update the dependant' do
+        dependant.reload
+        expect(dependant.in_full_time_education).to be_nil
+      end
+
+      it 'displays an error' do
+        expect(response.body).to match(I18n.t('activemodel.errors.models.dependant.attributes.in_full_time_education.blank_message', name: dependant.name))
```
Comment (Kalagan) [inline spec/requests/citizens/dependants/full_time_educations_spec.rb]: use `unescaped_response_body` here. `dependant.name` might have some special characters

File: app/services/flow/flows/citizen_dependants.rb (hunk 1 above)
Comment (reggieb) [reply app/services/flow/flows/citizen_dependants.rb]: Why? No matter how the question is answered, the next page is aways 'dependants_incomes'. The flow is only affected by the answer on leaving 'dependant_incomes'

Comment (Kalagan) [reply app/services/flow/flows/citizen_dependants.rb]: Ah yes sorry you're right.

Comment (reggieb) [reply app/services/flow/flows/citizen_dependants.rb]: I had to look at it a couple of times to get my head around the flow diagramme

Comment (Kalagan) [reply app/services/flow/flows/citizen_dependants.rb]: I just remembered that you can just write ` forward: :dependants_incomes` here

**Comment by reggieb** [comment]
There is a successful build on UAT at https://ap-724-add-dependant-in-full-t-applyforlegalaid-uat.apps.cloud-platform-live-0.k8s.integration.dsd.io

**Pull request merged**